virttest/storage.py

```diff
@@ -335,25 +337,40 @@ def save_to_file(self):
             fd.write(self.data)
 
 
-class ImageAccessInfo(object):
+class SingleImageAccessInfo(object):
```

Comment:
Looks like it doesn't need an extra concept. IMHO, what we do is to let `ImageAccessInfo` storing the number of `StorageAuth` from one to more. By achieving this, some backend specified data has to be stored within `StorageAuth`, such as iscsi initiators.